runtime/binding-mqtt/src/test/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/server/v5/ConnectionIT.java

```diff
@@ -414,6 +414,16 @@ public void shouldRejectWillRetainWithoutWillFlag() throws Exception
         k3po.finish();
     }
 
+    @Test
+    @Configuration("server.yaml")
+    @Specification({
+        "${net}/connect.max.packet.size.exceeded/client",
+        "${app}/connect.max.packet.size.exceeded/server"})
+    public void shouldIgnorePublishPacketBiggerThanMaxPacketSize() throws Exception
```

Comment:
Please name method for script scenario name, i.e. `shouldRejectMaxPacketSizeExceeded`.